### DISCUSSION TITLE
2-1. 출발하기 전 고민해야 할 두 가지

### DISCUSSION THREAD

**Discussion by imakerjun** [ssac-dev/yozm#6] [2장 똑똑하게 학습하고 싶어]
# 2-1. 출발하기 전 고민해야 할 두 가지 

## 1. 피드백의 부재로 내가 잘하고 있는지 판단하기 어려웠던 경험 한 가지 생각하기
외부의 피드백 없이 혼자 고민을 계속하다 보면 별 발전없이 같은 고민만 계속하게 될 수 있습니다. 이럴 때 터널 비젼에서 나를 꺼내줄 제3자가 필요합니다. 꺼피드백 없이 혼자 고민하며 막막했던 경험을 공유해주세요.

## 2. 즉각적인 피드백을 통해 무언가를 배울 수 있었던 사례 한 가지 생각하기
외부의 피드백을 통해 미쳐 난 보지 못했던 면을 발견하고 배우게 됩니다. 치면도색 양치질의 예와 같이, 즉각적인 피드백을 통해 무언가 배우고 성장한 경험을 공유해주세요.

## 3. 내가 사용하는 서비스 중 즉각적인 피드백을 제공하는 서비스 한 가지 공유하기
즉각적인 피드백이 사용성을 높이기 때문에 시중 여러 앱들은 최대한 빠르고 직관적인 피드백을 주려 노력합니다. 사용하고 있는 여러 서비스 중 알림과 같이 즉각적인 피드백을 제공하는 서비스를 공유해주세요.

**Comment by heeji289** [comment]
# 2-1. 출발하기 전 고민해야 할 두 가지
## 1. 피드백의 부재로 내가 잘하고 있는지 판단하기 어려웠던 경험 한 가지 생각하기

완전 혼자 고민한 적은 없다. 이전에 회사를 다니면서 동료 개발자들끼리 의사결정을 하고 그 내용을 구현하고 그랬었는데 그것에 대해 누군가가 그 방법이 옳아! 아니야! 말해주지는 않았던 것이 한 사례가 될 것 같다. (퇴사 한 지금 아쉬운 것은 기술적인 의사 결정에 대한 회고가 없었다는 것)

## 2. 즉각적인 피드백을 통해 무언가를 배울 수 있었던 사례 한 가지 생각하기

회사에 입사하고 한 달간 정말 많은.. 즉각적인 피드백을 통해 제품을 만들 때 고려해야하는 기본사항들을 빠르게 익힐 수 있었다. 혼자 학습할 때는 몰랐던 부분들을!

## 3. 내가 사용하는 서비스 중 즉각적인 피드백을 제공하는 서비스 한 가지 공유하기

X (트위터) 같은 경우, pull to refresh 시에 스피너를 통해 지금 최신 글을 불러오고 있어~ 알려주고 일정 시간 마다 팔로워의 새 글이 있으면 상단에 표시해준다!
